Libraries/Blob/RCTBlobManager.h

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright (c) 2015-2016 Silk Labs, Inc.
+ * All Rights Reserved.
+ */
```

Comment:
I'd love to keep the header here actually but probably should have the fb header :( Not sure how licensing works for RN.